netjsonconfig/backends/openwrt/converters/openvpn.py

```diff
@@ -11,7 +11,7 @@ def __intermediate_vpn(self, vpn):
             '.type': 'openvpn',
             'enabled': not vpn.pop('disabled', False)
         })
-        return super(OpenVpn, self).__intermediate_vpn(vpn, remove=[''])
```

Comment:
I see another occurrence of `super` in this file at line 23

Reply:
Done :smile: